src/components/Sidebar.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './sidebar.css';
+
+const Sidebar = () => (
+  <div className="nav-bar-container">
+    <div className="logo-container">
+      <h4>My-logo</h4>
+    </div>
+    <div className="nav-links-container">
+      <NavLink to="/">
+        HouseList
+      </NavLink>
+
+      <NavLink to="/add-house">
+        Add House
+      </NavLink>
+
+      <NavLink to="/reserve-list">
+        Reserve List
+      </NavLink>
+
+      <NavLink to="/my-reservations">
+        My Reservations
+      </NavLink>
+
+    </div>
+
+  </div>
+
+);
```

Comment:
- [ ] Please add these social media icons and copyright underneath as stated in your task description:
![image](https://user-images.githubusercontent.com/91486791/203977213-a16f4130-4dff-45e0-ba0f-e057380cdfe2.png)